OpenRA.Game/Primitives/ActionQueue.cs

```diff
@@ -67,10 +67,10 @@ int Index(DelayedAction action)
 
 	struct DelayedAction : IComparable<DelayedAction>
```

Comment:
We could make the struct an inner member of the `ActionQueue` which would reduce it's visibility (as well as possibly renaming it).